x-pack/plugins/event_log/server/event_log_start_service.test.ts

```diff
@@ -26,7 +26,9 @@ describe('EventLogClientService', () => {
 
       eventLogStartService.getClient(request);
 
-      expect(savedObjectsService.getScopedClient).toHaveBeenCalledWith(request);
+      expect(savedObjectsService.getScopedClient).toHaveBeenCalledWith(request, {
+        includedHiddenTypes: ['action', 'alert'],
+      });
```

Comment:
I think this approach means we won't have the alert/actions RBAC on the event log, but only the default SavedObjects RBAC. This means that if the user doesn't have privileges over the `action` and `alert` saved object types (as granted by _some_ plugin) this code will throw.
You'll have to explicitly grant privileges via the EventLog feature for this to work.

I think we need an approach where plugins can plug handlers into the Event Log.
For example, the Actions plugin could "tell" the EventLog how to handle `action` saved object types, and it will use the ActionsClient internally to actually get the object.
That way you'll enjoy RBAC and you won't have to explicitly have to add `actions` to the EventLog feature's privileges config.

Reply:
Yeah, that's basically the plan for how to fix issue: https://github.com/elastic/kibana/issues/63961 

Dang, I guess it means this code is throw away anyway (despite it being some one-liner-ish simple fixes to the existing code).  Probably makes sense to fix that issue in this PR as well.

Also forgot that I noticed the event log query should include the namespace, but doesn't currently.  I think this isn't critical for 7.9, will take a quick peek at fixing that here, but afraid more tests will be required / existing ones fixed, perhaps better as a separate issue:

https://github.com/elastic/kibana/blob/7ec48fd9662ad7c8898fe3d9162533f8dbafcfc7/x-pack/plugins/event_log/server/es/cluster_client_adapter.ts#L130-L135

Reply:
I haven't fixed this on the RBAC branch as it's broken on master and that PR is already way too big... but I _could_ make a quick follow up PR that uses the Alerts/Actions clients after that to fix master and that would mean that this PR can be used for the long term solution and doesn't *have* to be done by 7.9.

What do you think?

Reply:
I ended up spending 2 days on using the alerts/actionsClient to resolve the saved objects, waaay more complicated than I thought.  Going to have to punt on adding that here, which seems fine since RBAC for alerts got pushed as well.